### PR TITLE
Add the page in Japanese to the doc

### PR DESCRIPTION
**Pull request by ono-max**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/ja/docs/user-guide/secret-management.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+全てを Git で管理するのは便利ですが、Kubernetes の Secret や Terraform の Credential などといった機密データを Git に直接格納するのは安全ではありません。
+ここでは、そういった機密情報をどのように Git 上で安全に管理できるかについて説明します。
+
+基本的には以下のフローになっています。
+- PipeCD の Web コンソールで暗号化した機密情報を Git に格納
+- `Piped` は Deployment のタスクを実行する際に復号化
+
+## 前提条件
+
+この機能を使う為には RSA の公開鍵と秘密鍵のペアが必要になり、下記のコマンドで作ることができます。
```
Comment (nakabonne) [inline docs/content/ja/docs/user-guide/secret-management.md:18]: It doesn't have to be an RSA key pair, so let's remove it.
I opened a patch to fix the original: https://github.com/pipe-cd/pipe/pull/2891

File: docs/content/ja/docs/user-guide/secret-management.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+<br>
+
+![](/images/sealed-secret-encrypting-form.png)
+<p style="text-align: center;">
+暗号化された機密データの例
+</p>
+
+## 暗号化された機密データを Git へ格納する
+
+暗号化された機密データを利用できるようにする為には Application の `pipe.yaml` 内で指定する必要があります。
```
Comment (nakabonne) [inline docs/content/ja/docs/user-guide/secret-management.md:67]:
```suggestion
暗号化された機密データを利用できるようにする為には Application の `.pipe.yaml` 内で指定する必要があります。
```

**Comment by nakabonne** [comment]
@ono-max I appreciate you. I left a couple of comments.

**Comment by ono-max** [comment]
@nakabonne 

Hey, could you review this one?
Thanks!

**Comment by nakabonne** [comment]
Thank you!
/approve

**Pull request merged**